app/templates/agenda.html

```diff
@@ -7,39 +7,39 @@
 
 {% block content %}
 <form method="GET" action="/agenda">
-    <div class="mb-3 agenda_filters"> 
-        <label for="start_date">From</label><br>
-        <input class="filter" type="date" id="start_date" name="start_date" value={{ start_date }}><br>
-        <label for="end_date">To</label><br>
-        <input class="filter" type="date" id="end_date" name="end_date" value={{ end_date }}><br>
-        <input class="filter" type="submit" value="Get Agenda">
+    <div class="mb-3 agenda_filters">
+        <label for="start_date">{{ gettext("From") }}</label><br>
```

Comment:
Isn't jinja have a {% trans %} block or something? ^^"

Reply:
jinja2 does indeed have a trans block, but it also supports gettext. I'm not sure if using trans extracts the text strings to the .pot file.

Reply:
{% trans %} actually calls `gettext`. They use {% trans %} in their [examples](https://github.com/pallets/jinja/blob/master/examples/basic/translate.py).

Reply:
What I meant is that I'm not sure if the pybabel command to extract strings, does it from places that don't explicitly use gettext, but use trans. In any reate, https://jinja.palletsprojects.com/en/2.11.x/templates/#i18n has both as options, with a caveat that trans does not work for expressions unless those are bound first. Seems an overlay and needlessly complicated method of translating strings, if gettext (and my v1) do it without that.

Reply:
[Yes, it is possible](https://phrase.com/blog/posts/i18n-advantages-babel-python/).

About the bounding - I don't see why we should prefer:
```{{ gettext('Hello %(name)s!')|format(name=who|e) }}```

Or even:
```{{ gettext('Hello %(name)s!', name=who|e) }}```

Over:
```
{% trans name=who|e %}
   Hello, {{ name }}!
{% endtrans %}
```

The both pass the original variable to the template, the second one just seems more clean

Reply:
The code is none of the above, but `{{ gettext('Hello (name)', name=Yam) }}`. I don't see how that isn't clean.